tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/AspNet/AspNetMvc4Tests.cs

```diff
@@ -141,6 +145,50 @@ await Verifier.Verify(spans, settings)
                           .UseMethodName("_")
                           .UseTypeName(_testName);
         }
+
+        [SkippableFact]
+        [Trait("Category", "EndToEnd")]
+        [Trait("RunOnWindows", "True")]
+        [Trait("LoadFromGAC", "True")]
+        public async Task ClientDisconnect()
+        {
+            var isClassicMode = _testName.Contains(".Classic");
+
+            var data = $@"POST /ping HTTP/1.1
+Accept: application/json
+Content-Type: application/json
+Host: localhost:{_iisFixture.HttpPort}
+Content-Length: 25
+x-datadog-tracing-enabled: false
+User-Agent: testhelper
+Expect: 100-continue
+
+";
+
+            var testStart = DateTime.UtcNow;
+            Socket socket = new Socket(SocketType.Stream, ProtocolType.Tcp);
+            socket.Connect("localhost", _iisFixture.HttpPort);
+            socket.Send(Encoding.ASCII.GetBytes(data));
+            var bytes = new byte[100];
+            socket.Receive(bytes);
+            socket.Close();
+
+            Output.WriteLine($"[http] {Encoding.ASCII.GetString(bytes)}");
+            Encoding.ASCII.GetString(bytes).Should().StartWith("HTTP/1.1 100 Continue");
+
+            var spans = _iisFixture.Agent.WaitForSpans(
+                count: isClassicMode ? 0 : 1, // classic mode doesn't generate any spans in this scenario
+                minDateTime: testStart,
+                returnAllOperations: true);
+
+            ValidateIntegrationSpans(spans, expectedServiceName: "sample", isExternalSpan: false);
+
+            var settings = VerifyHelper.GetSpanVerifierSettings();
+
+            await Verifier.Verify(spans, settings)
```

Comment:
Is this version of the `Verify` okay to use? 
I ask because I think I ran into an issue with this one that Kevin found and pointed me to use the `VerifyHelper.VerifySpans` method instead as it sorted the spans where `Verifier.Verify` didn't.

Reply:
I just realized there is only a single span in the snapshots so sorting isn't an issue

Reply:
Yeah, you're absolutely right. It _will_ be ok in this case, but I think we should always use the `VerifyHelper` anyway where possible, so updated it, thanks!